### PR TITLE
Create sphinx_avg_query_wall

### PR DESCRIPTION
**Pull request by karelbarel**
works for sphinx 2.2.x. maybe for other versions to.

### CODE REVIEW

File: plugins/sphinx/sphinx_avg_query_wall [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  [sphinx_*]
+  env.category mysql
+
+=head1 AUTHOR
+
+martin fuxa <root@unihost.cz>
+
+=head1 LICENSE
+
+CC
```
Comment (ssm) [inline plugins/sphinx/sphinx_avg_query_wall]: This (CC) looks like a reference to "Creative Commons", where there are multiple licenses, none if which are abbreviated "CC" .  Could you specify or clarify the license?

**Comment by sumpfralle** [comment]
@karelbarel: are you still interested in this plugin? What do you think about the licence?

**Comment by sumpfralle** [comment]
@karelbarel: what do you think?

**Comment by sumpfralle** [comment]
closed due to inactivity

**Pull request closed without merging**